api/client/build.go

```diff
@@ -58,6 +58,7 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	flCPUSetCpus := cmd.String([]string{"-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 	flCPUSetMems := cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
 	flCgroupParent := cmd.String([]string{"-cgroup-parent"}, "", "Optional parent cgroup for the container")
+	flNetMode := cmd.String([]string{"-net"}, "default", "Connect a container to a network")
```

Comment:
Please change the option to `--network` to match `docker run`. Apparently it changed over there in June: https://github.com/docker/docker/commit/c0c7d5e71586ec8e4d54aef9e061f061e9223cc4